src/aiperf/accuracy/graders/code_execution.py

```
@@ -1,25 +1,214 @@
+            raise RuntimeError(_MISSING_LIGHTEVAL_HINT)
+
+    def extract_answer(self, response_text: str, **kwargs: Any) -> str:
+        """Return the code block lighteval would extract from the response.
+
+        ``extract_code`` looks for the last ```python ... ``` block in
+        the response and returns its body, or empty string when no
+        block is present.
+        """
+        try:
+            return str(extract_code(response_text) or "")
+        except Exception as exc:  # pragma: no cover - defensive
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)